crates/evm/core/src/backend/cow.rs

```diff
@@ -196,12 +196,12 @@ impl DatabaseExt for CowBackend<'_> {
 
     fn transact_from_tx(
         &mut self,
-        transaction: &TransactionRequest,
+        data: &Bytes,
         env: Env,
         journaled_state: &mut JournaledState,
         inspector: &mut dyn InspectorExt,
-    ) -> eyre::Result<()> {
-        self.backend_mut(&env).transact_from_tx(transaction, env, journaled_state, inspector)
+    ) -> eyre::Result<TransactionMaybeSigned> {
```

Comment:
maybe mention the reason for the signature change here as well, but yeah keep it short